python/ray/_raylet.pyx

```diff
@@ -929,12 +940,40 @@ cdef class CoreWorker:
                     CActorCreationOptions(
                         max_restarts, max_task_retries, max_concurrency,
                         c_resources, c_placement_resources,
-                        dynamic_worker_options, is_detached, name, is_asyncio),
+                        dynamic_worker_options, is_detached, name, is_asyncio,
+                        c_pair[CPlacementGroupID, int64_t](c_placement_group_id, bundle_index)),
                     extension_data,
                     &c_actor_id))
 
             return ActorID(c_actor_id.Binary())
 
+    def create_placement_group(
+                            self,
+                            c_string name,
+                            c_vector[unordered_map[c_string, double]] bundles,
+                            c_string strategy):
+        cdef:
+            CPlacementGroupID c_placement_group_id
+            CPlacementStrategy c_strategy
+        with self.profile_event(b"submit_task"):
+            if strategy == b"PACK":
+                c_strategy = PLACEMENT_STRATEGY_PACK
+            else:
```

Comment:
Can you check explicitly for SPREAD as well and raise an error if not that or pack?